pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -228,7 +228,10 @@ spec:
     - jiva-volume-create-getstoragepoolcr-default
     - jiva-volume-create-putreplicadeployment-default
     - jiva-volume-create-puttargetdeployment-default
+    - jiva-volume-create-listreplicapod-default
+    - jiva-volume-create-patchreplicadeployment-default
   output: jiva-volume-create-output-default
+  fallback: jiva-volume-delete-default
```

Comment:
What does fallback do here?
fallback is not same as rollback.
fallback is used to fallback to lower version.

Reply:
ok, what should be done, to delete the existing jiva resources. I had tested this and it worked.

Reply:
@AmitKumarDas should i use rollback ?